topics/request.md

```diff
@@ -1,316 +1,116 @@
-[//]: # (title: Request)
+[//]: # (title: Send a request)
 
-<include src="lib.md" include-id="outdated_warning"/>
+After adding the necessary [dependencies](client.md#add-dependencies) and [creating the client](client.md#create-client), you can send HTTP requests. You can configure the following request parameters:
+* Specify an HTTP method, such as `GET`, `POST`, `PUT`, and so on.
```

Comment:
Let's enumerate al of them. Also please let's stop using the constructs of "and so on.". It doesn't give users enough info.

Reply:
ok